<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cutting Stock Application</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout">
    <item>
     <widget class="QLabel" name="label_barras_disponiveis">
      <property name="text">
       <string>Barras disponíveis: 7, 8, 9, 10, 11, 12, 13, 15 metros</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_instrucao">
      <property name="text">
       <string>Digite os comprimentos das barras:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit_tamanho">
        <property name="placeholderText">
         <string>Comprimento</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_quantidade">
        <property name="placeholderText">
         <string>Quantidade</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_adicionar">
        <property name="text">
         <string>Adicionar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget_barras_desejadas">
      <column>
       <property name="text">
        <string>Tamanho</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Quantidade</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_resolver">
      <property name="text">
       <string>Resolver</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_barras_digitadas">
      <property name="text">
       <string>Barras digitadas:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_resultado">
      <property name="text">
       <string>Melhor combinação:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit_combinacoes"/>
    </item>
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
